src/app/features/notification/services/notification.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { NotificationComponent } from '@features/notification/components/notification.component';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class NotificationService {
+
+    constructor (private _snackBar: MatSnackBar) {
+    }
+
+    error (message: string, duration: number) {
+
+        this._snackBar.openFromComponent(NotificationComponent, {
+            data: message,
+            duration,
+            panelClass: 'snack-style-red',
+            verticalPosition: 'top'
+        });
+
+    }
+
+    showNotification (message: string, duration: number, color?: 'red' | 'green' | 'yellow') {
```

Comment:
names of methods should be consistent
above you are showing error with method `error`
here you are showing notification with method `showNotification`
do you see the difference?

you need to make both in the same format.
I prefer `showError`, `showNotification`, because `error` can be misunderstood and taken for property not a method